### ISSUE TITLE
Missing type map configuration or unsupported mapping

### ISSUE THREAD

**Issue by maurosampietro** [AutoMapper/AutoMapper#1674]
I'm trying to map a codefirst entityframework class back and forth its DTO:

```
public class MyDbEntity{
    public int Id{get;set;}
    public BindingList<OtherEntity>{get;set;}
}

public class MyDbEntityDto{
    public int Id{get;set;}
    public BindingList<OtherEntityDto>{get;set;}
}
```

I can map from EF to DTO but not back (DTO to EF) and when I try I get the exception in the subject.
Why is it happening? How can I workaround?

**Comment by lbargaoanu** [comment]
A repro would help. Make a gist that we can execute and see fail.

**Comment by maurosampietro** [comment]
I don't really know how to setup a repro with a db involved.
But, isn't my example minimal enough to make any hypothesis?

**Comment by lbargaoanu** [comment]
There is no need for a database, just the AutoMapper part. Well, the message should tell you what map is missing :)

**Comment by maurosampietro** [comment]
Ok, so i download the source code and looked into examples.
There's was one that was exactly my case.

I figured out the problem was how I was using you library.

I used mapper initializa multiple times like this (once for each map):

Mapper.Initialize( cfg => cfg.CreateMap<Dto1, Type1>() )
Mapper.Initialize( cfg => cfg.CreateMap<Dto2, Type2>() )
Mapper.Initialize( cfg => cfg.CreateMap<Dto3, Type3>() )

It turns out the only map configured it's the last one.
I thought it should sum up.

Maybe my misunderstanding is a common thing. 
It shouldn't have happened if i was forced to instantiate a mapper and initialize it by constructor.

Thank you
Bye
